Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -1011,6 +1011,11 @@ protected string DropdownItemId( int index )
         /// </summary>
         [Parameter] public int MinLength { get; set; } = 1;
 
+        /// <summary>
+        /// Specifies the maximum number of characters allowed in the input element.
+        /// </summary>
+        [Parameter] public int? MaxLength { get; set; }
```

Comment:
Maybe `MaxEntryLength`? That might make it clearer that in multi select mode this is referring to each individual selection rather than thinking of the autocomplete as always being one big input with a length. Of course in FreeTyping mode, with single select, it *is* one big input, but the name still makes sense because there's only one "entry."

Reply:
Maybe even more specific to avoid miss interpretations? MaxCharLength or MaxCharactersLength?

Reply:
As usual naming is hard we can spend hours over it 😂😂

Reply:
Indeed. FWIW, I still think MaxCharLength could lead someone to think it applies to the input as a whole, *even* in the case of multi select mode. But I agree there's no perfect answer. I defer to your judgement sir :)

Reply:
I like the `MaxEntryLength`. It correctly points to the correct element, IMO.